### DISCUSSION TITLE
Primeira homepage basicona!

### DISCUSSION THREAD

**Discussion by lucasrafaldini** [Data-Bureau/data-bureau.github.io#1] [Polls]
Fala, pessoal! Tudo bem?

Terminei esse primeiro rascunho para a nossa empreitada e acho que já está bom para um site "basico", somente enquanto referência de existência online para uma iniciativa que se dá no mundo real. 

Podemos usar o link nos materiais impressos e para compartilhar também, claro.

A questão que me intriga é: já está bom o bastante para pelo menos tirar do papel ou não?

**Comment by diaslais** [comment]
Ficou muito massa!
